csm/core/controllers/routes.py

```diff
@@ -28,7 +28,7 @@
 from csm.core.controllers.rgw.s3.users import (S3IAMUserListView, S3IAMUserView,
                                                S3IAMUserKeyView, S3IAMUserCapsView, S3IAMUserQuotaView)
 from csm.core.controllers.rgw.s3.bucket import S3BucketView
-
+from csm.core.controllers.version import VersionCompatibilityView
```

Comment:
Change this name as per the service class

Reply:
changed Controller view name from VersionCompatibilityView --> VersionInformationView